fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -51,7 +51,8 @@ var SESSION_TOKEN = {
   uaBrowserVersion : 'mock browser version',
   uaOS : 'mock OS',
   uaOSVersion : 'mock OS version',
-  uaDeviceType : 'mock device type'
+  uaDeviceType : 'mock device type',
+  tokenVerified: true
```

Comment:
I don't think so. If we set a `tokenVerificationId` for all tests, we risk attempting to insert duplicate primary keys if a test doesn't clean up after itself. For that reason, I wanted to force the tests to set their own `tokenVerificationId`, which can never clash.